app/models/float_field_type.rb

```diff
@@ -0,0 +1,53 @@
+class FloatFieldType < FieldType
+
+  attr_accessor :float
+ 
+  validates_numericality_of :float
+  validate :less_than, if: :validate_min?
+  validate :greater_than, if: :validate_max?
+
+  def validations=(validations_hash ={})
```

Comment:
This inherits from `FieldType`, which gives us the `validations` setter, so we don't actually need this